run

```diff
@@ -0,0 +1,197 @@
+#! /usr/bin/env bash
+
+set -euo pipefail
+
+# Define the versions of the two docker images
+django_image="ubuntudesign/django-app:v1.1.0"
+yarn_image="ubuntudesign/yarn:v0.1.0"
+
+# Usage instructions
+USAGE="Usage
+===
+
+  $ ./run \\
+    [-p|--port NUM]  # Specify local port for server \\
+    [--no-debug]     # Turn off Django's DEBUG setting \\
+    [COMMAND]        # Optionally provide a command to run
+
+If no COMMAND is provided, \`start\` will be run.
+
+Commands
+---
+
+- start [-w|--watch]: Run the development server (optionally watching for Sass changes) \\
+- watch: Run \`yarn run watch\`
+- build: Run \`yarn run build\`
+- test: Run \`yarn run test\` and \`./manage.py test\`
+- shell: Open a Django shell
```

Comment:
Similar to other comment, maybe we should be more explicit with the command naming for shell. It could be interpreted as a command line shell of the image, or we may possibly want that in the future.

Or possibly it is not needed, as `./run django shell` will do this already if I remember rightly.